cmd/plugin/cli/preflight.go

```diff
@@ -12,7 +12,13 @@ import (
 )
 
 const (
-	preflightSpec = "https://raw.githubusercontent.com/storageos/storageos.github.io/master/yaml/preflight.yaml"
+	// defaultCollectorImage can be removed once
+	// https://github.com/replicatedhq/troubleshoot/pull/392 merges, then it
+	// will default to "replicatedhq/troubleshoot:latest".
+	defaultCollectorImage = "storageos/troubleshoot:latest"
+
+	// defaultPreflightSpec will be used if not supplied by the user.
+	defaultPreflightSpec = "https://raw.githubusercontent.com/storageos/kubectl-plugin/main/specs/preflight.yaml"
```

Comment:
This points to the `latest`, I suggest to not do this.

Reply:
We'd like to be able to update the specs without releasing a new version of the plugin and getting users to install it. We currently keep the specs in the docs repo, but I think it would be better to keep everything in the same repo.

We could probably version this a little though, perhaps with a versioned directory, i.e. `specs/v2.5/preflight.yaml`.  WDYT?

Reply:
Versioned directory make sense for single file versioning. Go for it now and let's see how it works :)